api/src/main/java/org/pmiops/workbench/cdr/ConceptBigQueryService.java

```diff
@@ -51,29 +54,28 @@ public int getParticipantCountForConcepts(
         partitionSourceAndStandard.get(false).stream()
             .map(DbConceptSetConceptId::getConceptId)
             .collect(Collectors.toList());
-    StringBuilder innerSql = new StringBuilder("select count(distinct person_id) person_count\n");
-    innerSql.append("from ");
-    innerSql.append(String.format("`${projectId}.${dataSetId}.%s`", omopTable));
-    innerSql.append(" where ");
+    StringBuilder innerSql = new StringBuilder();
     ImmutableMap.Builder<String, QueryParameterValue> paramMap = ImmutableMap.builder();
     if (!standardList.isEmpty()) {
-      innerSql.append(conceptColumns.getStandardConceptColumn().name);
+      innerSql.append("select person_id\n");
+      innerSql.append("from `${projectId}.${dataSetId}.cb_search_all_events`\n");
+      innerSql.append("where is_standard = 1 and concept_id in ");
       generateParentChildLookupSql(
           innerSql, domain, "standardConceptIds", 1, standardList, paramMap);
+      innerSql.append("\n");
       if (!sourceList.isEmpty()) {
-        innerSql.append(" or ");
+        innerSql.append(" union all\n");
       }
     }
     if (!sourceList.isEmpty()) {
-      if (Domain.SURVEY.equals(domain)) {
-        innerSql.append("observation_source_concept_id");
-      } else {
-        innerSql.append(conceptColumns.getSourceConceptColumn().name);
-      }
+      innerSql.append("select person_id\n");
```

Comment:
Not required but we can have 
      innerSql.append("select person_id\n");
      innerSql.append("from `${projectId}.${dataSetId}.cb_search_all_events`\n");
      innerSql.append("where is_standard = {isStandard} and concept_id in ");
different method where we can pass isStandard bool

Reply:
Done